client/sidebar/sections/components/OmnichannelCallToggleReady.tsx

```diff
@@ -28,6 +28,13 @@ export const OmnichannelCallToggleReady = (): ReactElement => {
 	}, [voipClient]);
 	// TODO: move registration flow to context provider
 	const handleVoipCallStatusChange = useMutableCallback((): void => {
+		const { state } = voipClient.callerInfo;
+
+		if (['IN_CALL', 'OFFER_RECEIVED', 'ON_HOLD'].includes(state)) {
```

Comment:
pls remove the 'offer received' one from the list

Reply:
![image](https://user-images.githubusercontent.com/11577696/157748257-bf011b62-9fce-4e9b-8e95-7bc508853892.png)